src/app/shop/[category]/_components/TitleWrap/Filter.tsx

```diff
@@ -0,0 +1,105 @@
+'use client';
+
+import { InputField } from '@/components';
+import { FILTER_OPTIONS } from '@/constants/product';
+import { DeleteIcon, ResetIcon } from '@/public/index';
+import type { CategoryKey } from '@/types/Category';
+import classNames from 'classnames/bind';
+import { useState } from 'react';
+import styles from './Filter.module.scss';
+
+const cn = classNames.bind(styles);
+
+const DELETE_ITEM_COLOR = '#787878';
+
+export default function Filter({ category }: { category: CategoryKey }) {
+  const [selectedList, setSelectedList] = useState<string[]>([]);
+  const options = FILTER_OPTIONS[category];
+
+  const handleClickReset = () => {
+    setSelectedList([]);
+  };
+
+  const handleClickFilterItem = (item: string) => {
+    if (selectedList.includes(item)) {
+      setSelectedList(selectedList.filter((x) => x !== item));
+    } else {
+      setSelectedList([...selectedList, item]);
+    }
+  };
+
+  const handleRemoveItem = (item: string) => {
+    setSelectedList(selectedList.filter((x) => x !== item));
+  };
```

Comment:
사소하지만, x가 뭔가 이상한 것 같네요. 이런 식으로 하는 게 의미 전달이 더 잘 되지 않을까 싶습니다~
```suggestion
  const handleRemoveItem = (selectedItem: string) => {
    setSelectedList(selectedList.filter((item) => item !== selectedItem));
  };
```